AK/Assertions.cpp

```diff
@@ -21,6 +21,9 @@
 
 #if defined(AK_HAS_BACKTRACE_HEADER)
 #    include <cxxabi.h>
+#elif __cpp_lib_stacktrace >= 202011L
```

Comment:
always `#include <version>` before checking a feature test macro.

In the case that we check this in CMake though, we can just define a PRIVATE macro of AK_HAS_STD_BACKTRACE or similar.